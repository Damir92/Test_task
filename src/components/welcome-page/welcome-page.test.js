import React from 'react';
import {shallow} from 'enzyme';

import WelcomePage from './welcome-page';

const history = {
  push: () => {},
};

describe(`WelcomePage component`, () => {
  it(`WelcomePage snapshot`, () => {
    const tree = shallow(
        <WelcomePage
          history={history}
        />
    );

    expect(tree).toMatchSnapshot();
  });
});
