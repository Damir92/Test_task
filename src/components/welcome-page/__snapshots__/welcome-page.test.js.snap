// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomePage component WelcomePage snapshot 1`] = `
<Fragment>
  <WithStyles(CssBaseline) />
  <Styled(MuiBox)
    className="makeStyles-box-1"
    component="div"
  >
    <WithStyles(ForwardRef(Link))
      className="makeStyles-link-2"
      component="button"
      onClick={[Function]}
    >
      Show list
    </WithStyles(ForwardRef(Link))>
  </Styled(MuiBox)>
</Fragment>
`;
